scripts/google/covid_mobility/TestCovidMobility.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from covidmobility import covid_mobility, csv_row_to_obj
+
+
+# NOTE: More testing will be added soon.
+
+
+class TestStringMethods(unittest.TestCase):
+    def test1_output(self):
+        # Get the output file.
+        covid_mobility('./test1.csv')
```

Comment:
I would highly recommend passing in a output param here. We don't want to accidentally save the test output as our actual output.

Another recommendation is having a testdata/ subdirectory. Both in and exp output can go there. You can gitignore actual output or see if https://docs.python.org/2/library/stringio.html works.

Reply:
Ditto for other tests.

Reply:
Tests are now under **/tests/test***

tests/test1
tests/test2